features/step_definitions/buy_steps.js

```diff
@@ -0,0 +1,21 @@
+var {defineSupportCode} = require('cucumber');
+
+/* Sample_Syntax */
+/*
+defineSupportCode(function ({Given, When, Then}) {
+
+  Given(/^I want to buy some potatoes$/, function(){
```

Comment:
No entiendo la sintaxis que esta dentro del las palabras claves "Given, when, then".  En este caso,
 ¿Es lo mismo poner **Given(/^I want to buy some potatoes$/**  y **Given ('I want to buy some potatoes'**?

Reply:
ok perfecto,  usare esa expresión también en el código que desarrolle 👍